elasticsearch/elasticsearch.go

```diff
@@ -45,23 +48,26 @@ const (
 	esPort = "port"
 
 	invalidMetricType = "Invalid metric type found"
+	readDocErr        = "Cannot read the file"
```

Comment:
This is an unexported err that is referenced only once.  For readability I would move this to where the error is thrown and maybe include more than "cannot read the file".